modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java

```diff
@@ -364,14 +364,33 @@ protected void deleteLatestVersion(StagedModel stagedModel)
 	protected void exportImportPortlet(String portletId) throws Exception {
 		exportImportPortlet(
 			portletId, new LinkedHashMap<String, String[]>(),
-			new LinkedHashMap<String, String[]>());
+			new LinkedHashMap<String, String[]>(), true, false);
+	}
+
+	protected void exportImportPortlet(
+			String portletId, boolean staging, boolean exporting)
```

Comment:
Hi @joseabelenda , how about renaming `staging` to `portletStagingInProcess` and `exporting` to  `portletExportInProcess`? 
Also, can you order it alphabetically `(String portletId, boolean portletExportInProcess, boolean portletStagingInProcess)`? `portletId` must always come first

Reply:
Hi, @carolmariaabb.

I've reviewed and the variable _exporting_ isn't necessary.
I renamed _staging_ to _portletStagingInProcess_.